File: sentimental-mario-less/mario.py
# TODO
from cs50 import get_int


def main():

    # assigns return value of get_height to heigh variable
    height = get_height()
    i = 1
    j = height - 1

    # prints pyramid
    while height >= i:
        print(" " * j, end="")
        print("#" * i)
        i += 1
        j -= 1


def get_height():

    # prompts user for integer input (need to implement between 1 and 8)
    while True:
        n = get_int("Height: ")
        if n < 9 and n > 0:
            break
    return n


main()

File: mario-less/mario.c
#include <stdio.h>
#include <cs50.h>

int main(void)
{
    int n;
    int k = 0;
/*Takes integer input for numbers between 1 and 8, re-prompts if not */
    do
    {
        n = get_int("Size: ");
    }
    while (n < 1 || n > 8);

    int count = n;

    while (n > k)
    {
        for (int l = count; l > 1; l--)
        {
            printf(" ");
        }
        for (int j = 0; j <= k; j++)
        {
            printf("#");
        }
        k++;
        count--;
        printf("\n");
    }
}
